src/packcc.c

```diff
@@ -1432,95 +1432,137 @@ static void verify_captures(context_t *ctx, node_t *node, node_const_array_t *ca
     }
 }
 
-static void dump_node(context_t *ctx, const node_t *node) {
+void dump_escaped(const char* s) {
+    char buf[5];
+    if (s == NULL) {
+        fprintf(stdout, "null");
+        return;
+    }
+    while(*s) {
+        escape_character(*s++, &buf);
+        fprintf(stdout, "%s", buf);
+    }
+}
+
+void dump_void_value(ullong_t value) {
```

Comment:
It's better to add a `static` modifier to make this function hidden under file-scope.